superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/CollapsibleControl.tsx

```diff
@@ -78,7 +84,12 @@ const CollapsibleControl = (props: CollapsibleControlProps) => {
           onChange(value);
         }}
       >
-        {title}
+        <>
+          {title}{' '}
```

Comment:
maybe better use `&nbsp;` instead ' '?

Reply:
I had it like that, but noticed it was done as `{' '}` in another similar place, so opted for that. I'll change it in both places.